charts/github-actions-runners/values.yaml

```diff
@@ -0,0 +1,36 @@
+runnerDeployment:
+  enabled: true
+  # -- Declare if the Runner Image is in a private repo
+  isRepositoryPrivate: true
```

Comment:
In all of our other charts this has been nested into "image" - given that you also have repositoryName in this level it could be confusing for an operator - would recommend putting this into "image"

Reply:
Also should this not be "false" by default given that this is a public image by default?

Reply:
Corrected